docs/source/collections.rst

```diff
@@ -0,0 +1,12 @@
+Private Collections Enhancements
+================================
+
+Private Collections are not supported a Distributed Content Addressable Storage type (or DCAS)
```

Comment:
May be we need to re-word this sentence. It's confusing.

Reply:
done